tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -708,6 +715,17 @@ describe(__filename, () => {
 
     expect(root.find(Notice)).toHaveLength(1);
     expect(root.find(Notice).children()).toHaveText('Added to collection');
+
+    let state = root.state();
+    expect(state.addonAddedStatus).toEqual(ADDON_ADDED_STATUS_SUCCESS);
+
+    // After 5 seconds the Notice will go away.
+    clock.tick(5010);
+
+    state = root.state();
+    expect(state.addonAddedStatus).toEqual(null);
+    root.setProps({ hasAddonBeenAdded: true });
```

Comment:
I'm not sure why you're setting the state here again, and I would expect odd results because of this. Was this just a line of code left in by accident?

Reply:
that's what I thought too but it seems that I need to get the updated state (when I do it this way it works :/) 
I also had to setProps too which I thought was odd but again it worked this way. Is it possible this is happening b/c i am missing some other piece of code here?